templates/command/new/headersOnly/bitbucket-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+clone:
+  depth: full
+
+definitions:
+  steps:
+    - step: &Versioning
+        name: Versioning
+        script:
+          - pipe: docker://hemelectronics/pipe-gitflow-versioning:0.1.3
+            variables:
+              BITBUCKET_USERNAME: "${BITBUCKET_USERNAME}"
+              BITBUCKET_PASSWORD: "${BITBUCKET_PASSWORD}"
+        artifacts:
+          - .version/*
+        services:
+          - docker
+        caches:
+          - docker
+    - step: &CreatePackage
+        name: Build
+        image:
+          name: hemelectronics/buildcpp-native-gcc:3.0.0
+          run-as-user: 1000
+        script:
+          - echo "Build running"
+          - CHANNEL=$(python3 ./getChannel.py)
+          - conan create . hem/"$CHANNEL"
+          - VERSION=`cat .version/Version.txt`
+          - conan upload {{ name }}/"$VERSION" --all -r hem --confirm
+        artifacts:
+          - build/bin/{{ name }}
```

Comment:
jak budujesz paczkę conanową to te artefakty nie mają sensu

Reply:
no a co jeśli tak na szybko na brudno chcesz sobie pobrać tą paczkę?